code/modules/client/preferences/prosthetic.dm

```diff
@@ -0,0 +1,16 @@
+/datum/preference/choiced/prosthetic
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_key = "prosthetic"
+	savefile_identifier = PREFERENCE_CHARACTER
+
+/datum/preference/choiced/prosthetic/init_possible_values()
+	return list("Random") + GLOB.limb_choice
+
+/datum/preference/choiced/prosthetic/is_accessible(datum/preferences/preferences)
+	if (!..(preferences))
```

Comment:
I noticed that every other quirk seems to do this, but is this necessary?
Seems like this would read better and be more consistent with the rest of our code as:
```suggestion
	. = ..()
	if (!.)
```